### ISSUE TITLE
how to calculate the accuracy of training and validation

### ISSUE THREAD

**Issue by alimsvn** [akTwelve/tutorials#5]
Hi;
Thank you for your very useful training. I've implemented your model and it worked nice. Now I want to calculate the accuracy of the model in train , val and test data but when I add the ['accuracy'] parameter to the model in model.py it doesn't work. Is there any way to get the accuracy?
Thanks.

**Comment by akTwelve** [comment]
Hi @alimsvn, sorry, this email notification ended up in my spam folder. Not sure how you're calculating accuracy in model.py, but one potential issue is that Jupyter notebooks don’t automatically reload modules unless you add a couple lines to the top of your notebook:
%load_ext autoreload # load the extension
%autoreload 2 # autoreload all modules

If that doesn't work, can you share your modification to model.py?

**Comment by alimsvn** [comment]
Hi Adam, and thank you for the answer. I added  metrics=["accuracy"] to the line 2186 :
self.keras_model.compile(
            optimizer=optimizer,
            loss=[None] * len(self.keras_model.outputs))
but nothing happened and I also tested this :
new_metrics = ["accuracy"]
self.keras_model.metrics_tensors.append(new_metrics)
in model.py like what's in line 2199 for "loss" but it didn't work too, now I understand from your point that I have to write a function which calculates the accuracy and then call it from somewhere in model.py

**Comment by akTwelve** [comment]
I think you will also need to add it to the "results.append({...})" part (line 2532) that gets returned by the "detect()" method. I haven't done this before, but just by following the code, it seems like the best place to insert it.
